tests/test_bears/TestBear.py

```diff
@@ -1,5 +1,5 @@
 from coalib.bears.LocalBear import LocalBear
-from coalib.results.Result import Result
+from coalib.results.Result import Result  # Ignore PyUnusedCodeBear
```

Comment:
I don't think we even need this import..... Remove and see if anything(including tests) breaks.